scripts/kind-e2e/lib_armada_deploy_kind.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# This should only be sourced
+if [ "${0##*/}" = "lib_helm_deploy_subm.sh" ]; then
+    echo "Don't run me, source me" >&2
+    exit 1
+fi
+
+### Variables ###
+
+# FIXME: Armada support for getting these values, vs hardcoding?
+declare -A cluster_CIDRs=( ["cluster1"]="10.4.0.0/14" ["cluster2"]="10.8.0.0/14" ["cluster3"]="10.12.0.0/14" )
+declare -A service_CIDRs=( ["cluster1"]="100.1.0.0/16" ["cluster2"]="100.2.0.0/16" ["cluster3"]="100.3.0.0/16" )
```

Comment:
FWIR, you can query the cluster for the CIDRs via `kubectl cluster-info dump`.

Reply:
I see pod_CIDRs there for each node ("podCIDR": "10.8.0.0/24"), which seem to be related but not the same as the cluster_CIDRs here. I also see the cluster and service CIDR vars ({ "name": "SUBMARINER_CLUSTERCIDR", "value": "10.8.0.0/14" }, { "name": "SUBMARINER_SERVICECIDR", "value": "100.2.0.0/16" }) that are a result of these being used by Helm, but that of course is a chicken-and-egg situation. Am I missing something else, or can you be more specific about what you have in mind? Thanks!

Reply:
Tracking with https://github.com/submariner-io/submariner/issues/372.